classes/submission/action/EditorAction.inc.php

```diff
@@ -198,7 +198,7 @@ public function addReviewer($request, $submission, $reviewerId, &$reviewRound, $
                         PKPNotification::NOTIFICATION_TYPE_ERROR,
                         ['contents' => __('email.compose.error')]
                     );
-                    trigger_error($e->getMessage(), E_USER_WARNING);
+                    trigger_error('Failed to send email: ' . $e->getMessage(), E_USER_WARNING);
```

Comment:
This is just to address a strange/unhelpful error message in the error_log while running the tests.